x-pack/plugins/ingest_pipelines/public/application/components/pipeline_editor/components/processor_form/field_components/drag_and_drop_text_list.scss

```diff
@@ -21,4 +17,12 @@
   &__labelContainer {
     margin-bottom: $euiSizeXS;
   }
+}
+
+.pipelineProcessorsEditor__form__dragAndDropList__grabIcon {
+  height: $euiSizeXL;
+  width: $euiSizeXL;
+  display: flex;
+  align-items: center;
+  justify-content: center;
 }
```

Comment:
Or you can just add padding since you already know it's always going to be a compressed input with a small icon.

```suggestion
.pipelineProcessorsEditor__form__dragAndDropList__grabIcon {
  padding-top: $euiSizeS;
} 
```

Reply:
Here's how it looks with just the padding:

![image](https://user-images.githubusercontent.com/1238659/181404947-e85ddd90-5ad4-4723-8c28-3d30c361f49f.png)

It gets a little cramped on either side. I think I'd prefer to leave it as-is for symmetry with the EuiIButtonIcon on the other side.

Reply:
Ah yeah, my example assumed you were using `gutterSize`.